simulation_parameters/microbe_stage/organelles.json

```diff
@@ -308,6 +308,41 @@
         "editorButtonGroup": "Organelle",
         "editorButtonOrder": 7
     },
+    "thermosynthase": {
+        "mpCost": 50,
+        "initialComposition": {
+            "ammonia": 1,
+            "phosphates": 1
+        },
+        "hexes": [
+            {
+                "r": 0,
+                "q": 0
+            }
+        ],
+        "processes": {
+            "bacterial_thermosynthesis": 1
+        },
+        "components": {
+            "storage": {
+                "capacity": 0.5
+            },
+            "processor": {
+                "colourChangeFactor": 1
+            }
```

Comment:
Does some other organelle specify this? As far as I remember and just quickly checked, this kind of component doesn't exist, so I wouldn't specify it here for this new organelle.

Reply:
Almost all of them do. You're right that it doesn't look like it's actually used, but that sounds like a problem for another PR.

Reply:
Fine, https://github.com/Revolutionary-Games/Thrive/issues/3476